docs/public/s2i-to-devfile.adoc

```diff
@@ -0,0 +1,92 @@
+== Migrating from S2I components to Devfile components
+
+Devfiles are new way of deploying component with odo that offers more convenience and flexibility. Read more about devfiles https://devfile.github.io/[here]
+
+Before 2.0 release odo deploys component with a different mechanism called as s2i based components. The newer way of deploying components is with devfiles.
+
+Exisiting users using s2i based components could migrate to devfile component by following the below steps:
+
+. Change the current directory to the component directory you want to migrate.
++
+[source,sh]
+----
+ $ cd <directory-name>
+----
+
+. Check the component status 
++
+[source,sh]
+----
+ $ odo list
+
+Openshift Components: 
+APP     NAME      PROJECT        TYPE       SOURCETYPE     STATE
+app     myapp     default        nodejs     local          Pushed
+
+----
+
+. Convert s2i component to devfile component, it would convert the s2i based configuration files to devfile configuration files.
```

Comment:
S2I needs to be capitalized

Reply:
Add `:` to end of sentence, not a period

Reply:
This may be better: `Convert a S2I component to devfile by running the following command:` maybe?